img/c/cognitoforms.com.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="900" height="900" version="1.0"><path fill="#D85326" d="M420.5 131.1c-6.6.5-12.3 1.3-12.7 1.7-.4.4-4.6 17.1-9.3 37.2-4.7 20.1-8.9 37.1-9.3 37.9-.5.8-4.7 2.5-9.5 3.8-12.5 3.3-30 10.3-45.3 18.2l-13.2 6.8-17.9-11c-9.8-6.1-24.6-15.2-33-20.4-8.3-5.1-15.6-9.3-16.2-9.3-1.3 0-18.4 14.6-29.4 25.1-4.2 4.1-12.8 13.2-19.1 20.3l-11.4 13 5.1 8c2.7 4.5 12.2 19.5 20.9 33.5l16 25.4-2.7 4.6c-8.7 15.1-19.4 40.7-24 57.3-1.4 5.1-1.8 5.7-4.8 6.3-1.7.4-17.4 4-34.7 8-17.3 4.1-33.2 7.7-35.2 8-3.6.7-3.7.8-4.5 6.4-1.1 7.7-.8 71.9.4 75.5l.9 3 37.8 8.7c20.7 4.8 38 9 38.3 9.3.4.4 1.7 4.3 3 8.7 4.9 17.1 15 40.9 22.4 52.9l3.1 5-2.6 4.3c-1.3 2.3-8.6 13.9-16 25.7-7.4 11.8-15.5 24.8-18 28.8s-4.6 7.7-4.6 8.2c0 3.4 36 40.5 51.7 53.3 6.5 5.3 7.4 5.7 9.5 4.6 1.3-.6 16.4-9.9 33.6-20.5 17.1-10.7 31.4-19.4 31.6-19.4.3 0 3.3 1.6 6.7 3.6 13.3 7.8 35.3 16.8 54.7 22.4 4.7 1.4 7.2 2.6 7.2 3.6 0 3.6 17.2 73.9 18.3 74.6.7.4 2.6.8 4.2.8 1.7.1 10.4.5 19.5 1 14.8.8 51.5-.1 57.6-1.5 1.8-.4 3-4.7 11-38.5l8.9-38 7-2.1c18.2-5.2 41.9-14.9 54.3-22.2 3.5-2 6.5-3.7 6.7-3.7.2 0 15.3 9.2 33.5 20.5s33.4 20.5 33.7 20.5c4.1 0 59.3-54.8 59.3-58.9 0-.4-9.3-15.6-20.6-33.7l-20.6-32.9 3.7-6.5c8.8-15.7 17.1-35.4 21.8-52.1 2.9-10-.8-8.2 35.2-16.5 13.8-3.1 29.3-6.7 34.5-7.9l9.5-2.2.9-3.9c1.2-4.8 1.4-66.7.3-74.5-.8-5.6-.9-5.7-4.5-6.4-6-1-72.6-16.7-73.4-17.2-.5-.2-.8-1.2-.8-2.1 0-.9-2.3-8.3-5-16.5-4.8-13.9-12-30.3-19-42.8l-3.2-5.7 20.1-32.1c11.1-17.7 20.4-32.8 20.7-33.6 1.8-4.5-54.6-60.6-59.5-59.2-.9.2-16.2 9.4-33.9 20.4l-32.4 20-13.1-6.8c-17.6-9-34.9-15.7-51.2-19.8l-4-1.1-7.4-31.7c-4-17.5-8.1-34.7-9-38.3l-1.6-6.5-5.5-.7c-10-1.2-53.3-1.7-65.5-.7zM474 269.6c38.7 5.4 73.4 23.4 102 53 11.6 12 18 20.6 25.9 34.4 5.3 9.2 13.7 27.8 12.9 28.5-.2.1-8.4 2.2-18.3 4.8-9.9 2.5-29.7 7.6-44 11.3-14.3 3.7-26.9 6.9-28 7.1-1.5.3-2.4-.4-3.3-2.7-4.5-10.2-18.3-25.1-29.5-31.9-13.8-8.2-26-11.5-42.7-11.4-15 .1-24 2.2-37.5 8.7-42.5 20.8-57.1 72.3-32.3 114.4 5 8.5 18.9 22 28.3 27.6C429 526 455 528 479.2 518.8c18.1-6.9 31.6-19.2 44.4-40.4 1-1.7 4-1 46 10.1 24.7 6.5 45.3 12.2 45.8 12.6 2.4 2.1-12.3 30.7-23.9 46.5-7.9 10.8-28 30.5-39.5 38.6-20.1 14.2-46.4 25.2-71.7 30-16.9 3.2-45.7 3.2-62.5 0C321.3 598 258.3 508.4 274.9 413c12.5-71.9 71.8-129.9 145.6-142.5 19.8-3.4 34.1-3.6 53.5-.9z"/></svg>
```

Comment:
Small update removing the padding around the icon
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 639.8 635.7"><path fill="#d85326" d="M290.9.5a66 66 0 0 0-12.7 1.7c-.4.4-4.6 17.1-9.3 37.2-4.7 20.1-9 37.1-9.3 38-.5.7-4.7 2.4-9.5 3.7-12.5 3.3-30 10.3-45.3 18.2l-13.2 6.8-18-11-33-20.4a189 189 0 0 0-16.1-9.3c-1.3 0-18.4 14.6-29.4 25.1A487.5 487.5 0 0 0 76 110.8l-11.4 13 5 8 21 33.5 16 25.4-2.7 4.6a295.8 295.8 0 0 0-24 57.3c-1.4 5.1-1.8 5.7-4.8 6.3l-34.7 8L5.2 275c-3.6.7-3.7.8-4.5 6.4-1.1 7.7-.8 72 .4 75.5l.9 3 37.8 8.7c20.7 4.8 38 9 38.3 9.3.4.4 1.7 4.3 3 8.7 4.9 17.1 15 41 22.4 53l3 5-2.5 4.2c-1.3 2.3-8.6 14-16 25.7l-18 28.8c-2.5 4-4.6 7.7-4.6 8.2 0 3.4 36 40.5 51.7 53.3 6.5 5.3 7.4 5.7 9.5 4.6 1.3-.6 16.4-9.9 33.6-20.5a1167 1167 0 0 1 31.6-19.4 267.7 267.7 0 0 0 61.4 26c4.7 1.4 7.2 2.6 7.2 3.6 0 3.6 17.2 74 18.3 74.6.7.4 2.6.8 4.2.8l19.5 1A451 451 0 0 0 360 634c1.8-.4 3-4.7 11-38.5l8.9-38 7-2a277 277 0 0 0 61-26c.2 0 15.3 9.2 33.5 20.5s33.4 20.5 33.7 20.5c4 0 59.3-54.8 59.3-58.9 0-.4-9.3-15.6-20.6-33.7L533.2 445l3.7-6.5a255.8 255.8 0 0 0 21.8-52c2.9-10-.8-8.3 35.2-16.6 13.8-3 29.3-6.7 34.5-7.9l9.5-2.2.9-3.9a800 800 0 0 0 .3-74.5c-.8-5.6-1-5.7-4.5-6.4-6-1-72.6-16.7-73.4-17.2-.5-.2-.8-1.2-.8-2a284 284 0 0 0-24-59.3l-3.2-5.8 20-32A1342 1342 0 0 0 574 125c1.8-4.5-54.6-60.6-59.5-59.2a756 756 0 0 0-34 20.4l-32.3 20-13.1-6.8c-17.6-9-35-15.7-51.2-19.8l-4-1-7.4-31.8a2952 2952 0 0 0-9-38.3L361.9 2l-5.5-.7C346.4 0 303-.5 290.9.5Z"/><path fill="#fff" d="M344.4 139a173.4 173.4 0 0 1 102 53c11.6 12 18 20.6 25.9 34.4 5.3 9.2 13.7 27.8 12.9 28.5l-18.3 4.8a10088.5 10088.5 0 0 0-72 18.4c-1.5.3-2.4-.4-3.3-2.7a88.3 88.3 0 0 0-29.5-31.9 75.9 75.9 0 0 0-42.7-11.4c-15 .1-24 2.2-37.5 8.7-42.5 20.8-57.1 72.3-32.3 114.4 5 8.5 18.9 22 28.3 27.6a82.6 82.6 0 0 0 71.7 5.4c18-6.9 31.6-19.2 44.4-40.4 1-1.7 4-1 46 10.1a1182 1182 0 0 1 45.8 12.6c2.4 2.1-12.3 30.7-24 46.5-7.8 10.8-28 30.5-39.4 38.6a190.5 190.5 0 0 1-71.7 30 204.7 204.7 0 0 1-62.5 0 175.9 175.9 0 0 1-143-203.2c12.7-71.9 72-129.9 145.8-142.4 19.8-3.4 34-3.6 53.5-.9"/></svg>
```

Reply:
Thank you very much!